ui/src/apps/keyvis/RootComponent.less

```diff
@@ -135,3 +150,16 @@
     height: 100%;
   }
 }
+
+#brightness-overlay {
+  background-color: white;
+  padding: 8px;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  box-shadow: 0 100px 80px rgba(0, 0, 0, 0.12);
```

Comment:
Prefer to use antd variables like `box-shadow: @box-shadow-base`:

https://github.com/ant-design/ant-design/blob/master/components/menu/style/index.less#L17

You can import these antd variables by adding `@import '~antd/es/style/themes/default.less';`.

Reply:
Done!